chore: add changelog for the fuzzy search API fix (#9745)
emqx/emqx: changes/v5.0.15/fix-9726-en.md

> Fuzzy search API results were missing any information which could tell clients if more results are available on next pages, this is now fixed by providing `hasnext` flag in responses.

Reviewer: ```suggestion
Client fuzzy search API results were missing information which could tell if more results are available in the next pages, this is now fixed by providing `hasnext` flag in the response.
```
